docs/core/runtime-config/threading.md

```diff
@@ -94,6 +94,44 @@ Project file:
 </Project>
 ```
 
+## Windows thread pool
+
+- For projects on Windows, configures whether the thread management is delegated to the Windows thread pool.
+- If you omit this setting or the platform is not Windows, the managed (portable) thread pool is used instead.
+- Only applications published with Native AOT on Windows use the Windows thread pool by default, for which you can opt to use the managed thread pool instead by disabling the config setting.
+
+| | Setting name | Values |
```

Comment:
Would be good to add a "Version introduced" column like in some of the newer tables below.